mono/utils/mono-threads.c

```diff
@@ -48,7 +48,7 @@ The GC has to acquire this lock before starting a STW to make sure
 a runtime suspend won't make it wronly see a thread in a safepoint
 when it is in fact not.
 */
-static MonoCoopSem global_suspend_semaphore;
+static MonoSemType global_suspend_semaphore;
```

Comment:
Could you document, for future references, why this semaphore cannot be a MonoCoopSem?